### PR TITLE
Adding Eureka Service for Client Discovery

### PR DESCRIPTION
**Pull request by Yash-VIT**

### CODE REVIEW

File: backend/cloud-gateway/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import java.net.*;
+import java.io.*;
+import java.nio.channels.*;
+import java.util.Properties;
+
+public class MavenWrapperDownloader {
```
Comment (sankalpjain99) [inline backend/cloud-gateway/.mvn/wrapper/MavenWrapperDownloader.java:21]: add this is gitignore

File: backend/cloud-gateway/.mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.1/apache-maven-3.8.1-bin.zip
```
Comment (sankalpjain99) [inline backend/cloud-gateway/.mvn/wrapper/maven-wrapper.properties:1]: add this is gitignore

File: backend/cloud-gateway/mvnw [hunk 3]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```
Comment (sankalpjain99) [inline backend/cloud-gateway/mvnw:4]: add this is gitignore

File: backend/cloud-gateway/mvnw.cmd [hunk 4]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
+@REM Licensed to the Apache Software Foundation (ASF) under one
+@REM or more contributor license agreements.  See the NOTICE file
+@REM distributed with this work for additional information
+@REM regarding copyright ownership.  The ASF licenses this file
```
Comment (sankalpjain99) [inline backend/cloud-gateway/mvnw.cmd:5]: add this is gitignore

**Review by sankalpjain99 — CHANGES_REQUESTED** [review]
Some changes are needed

File: backend/cloud-gateway/mvnw.cmd (hunk 4 above)
Comment (Yash-VIT) [reply backend/cloud-gateway/mvnw.cmd:5]: Done.

**Pull request merged**